src/main/java/formflow/library/ScreenController.java

```diff
@@ -104,6 +104,10 @@ ModelAndView getScreen(
       submission.setUrlParams(query_params);
     }
 
+    if (uuid != null) {
+      updateIterationIsCompleteMarker(flow, uuid, submission, currentScreen);
+    }
+
```

Comment:
This is an attempt to update the marker when the SNE uses a GET to get to the end of a subflow. Quick testing is showing me it doesn't work, but I haven't debugged it yet. I half suspect that UUID is not included here, which maybe means that the check needs to be in a different spot??? Not sure.  

There is some code in the starter app that I was using to mimic the flow that this needs to work for:   https://github.com/codeforamerica/form-flow-starter-app/tree/iteration_is_complete_%23185779545